Gemfile

```diff
@@ -14,7 +14,8 @@ gem 'seamless_database_pool'
 
 gem 'le', '~> 2.2'
 gem 'os'
-gem 'redis', '~> 3.1.0'
+gem 'redis', '~> 3.3.1'
+gem 'redis-slave-read', require: false, github: 'islemaster/redis-slave-read', ref: 'master'
```

Comment:
Worth adding a comment in the `Gemfile` linking to the PR, so the rationale for the github fork is clear and we know when it can be replaced with the upstream gem?